rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -102,6 +109,21 @@ void Player::play(const PlayOptions & options)
 
   wait_for_filled_queue(options);
 
+  rosbag2_transport_->configure();
+  // If playback should not be paused in the beginning, activate lifecycle node right away.
+  // If playback should be paused, move on.
+  // Playback will detect when node is activated, either from here or externally.
+  if (!options.paused) {
+    rosbag2_transport_->activate();
+  } else {
+    paused_ = true;
```

Comment:
do we really need that `paused_` variable for bookkeeping? Can't we rely on the state of the lifecycle node?

Reply:
Oh yeah you're right. I messed up somewhere. 9c92972